### PR TITLE
feat(rdfjs/fetch-lite): allow body

### PR DESCRIPTION
**Pull request by tpluscode**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://github.com/rdfjs-base/fetch-lite?tab=readme-ov-file#options> (see `body`)
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

File: types/rdfjs__fetch-lite/index.d.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import { Formats } from "@rdfjs/formats";
 import { BaseQuad, DatasetCore, DatasetCoreFactory, Quad, Stream } from "@rdfjs/types";
 
-export interface FormatsInit extends RequestInit {
-    formats: Formats;
+export interface FormatsInit extends Omit<RequestInit, "body"> {
+    formats?: Formats;
```
Comment (tpluscode) [inline types/rdfjs__fetch-lite/index.d.ts:5]: Contrary to the packages's readme, `formats` are not required directly but can be provided by the RDF/JS environment: https://github.com/rdfjs-base/fetch-lite/blob/master/Factory.js#L9

Comment (jeswr) [reply types/rdfjs__fetch-lite/index.d.ts:5]: > Contrary to the packages's readme, formats are not required directly but can be provided by the RDF/JS environment: https://github.com/rdfjs-base/fetch-lite/blob/master/Factory.js#L9

Can you PR a change to the readme as well - happy to approve this one @bergos signs off on the change in the docs.

Comment (tpluscode) [reply types/rdfjs__fetch-lite/index.d.ts:5]: I cannot. I'm banned from all Thomas' repositories

**Review by jeswr — APPROVED** [review]

**Comment by tpluscode** [comment]
Ready to merge

**Pull request merged**